pkgs/tools/misc/xsel/default.nix

```diff
@@ -1,13 +1,22 @@
-{stdenv, fetchurl, xlibsWrapper}:
+{stdenv, fetchFromGitHub, xlibsWrapper, autoreconfHook }:
 
-stdenv.mkDerivation {
-  name = "xsel-1.2.0";
-  src = fetchurl {
-    url = http://www.vergenet.net/~conrad/software/xsel/download/xsel-1.2.0.tar.gz;
-    sha256 = "070lbcpw77j143jrbkh0y1v10ppn1jwmjf92800w7x42vh4cw9xr";
+stdenv.mkDerivation rec {
+  name = "xsel-${version}";
+
+  version = "git-2016-09-02";
```

Comment:
Just consulted `doc/coding-conventions.xml`, looks like what I want is `"xsel-unstable-2016-09-02"` as the full name, which probably means I should have name be `"xsel-unstable"` and version be `"2016-09-02"`, since that's how nix would parse the full name.

>  If a package is not a release but a commit from a repository, then the version part of the name _must_ be the date of that (fetched) commit. The date must be in `"YYYY-MM-DD"` format.
> Also append <literal>"unstable"</literal> to the name - e.g., `"pkgname-unstable-2014-09-23"`.